apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,7 +157,3 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
-
-.conversation-view .panel {
```

Comment:
not sure, I think it's used in views/conversation/index.html :) `git grep conversation-view` is your friend